src/layout/CommunitySection/CommunitySection.svelte

```diff
@@ -51,7 +53,7 @@
 							{#each Array(35) as _}
 								<Contributor
 									html_url="https://github.com/yaichenbaum"
-									icon={Profile}
+									avatar_url="data:image/svg+xml;{Profile}"
```

Comment:
Is this intentional?

Reply:
Yes. When i converted to Fluent Svelte, I forgot that `<PersonPicture>` doesn't support the icon attribute, so this inserts the raw SVG into a `data:` URI.